src/components/salesEvidenceSearch.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ChangeEvent } from 'react'
+import { IntersectedSearchAndFilterParams } from '../types'
+import { SearchWidget } from './SearchWidget'
+import DatePicker from 'react-datepicker'
+import 'react-datepicker/dist/react-datepicker.css'
+import { SaleTypeDropdown } from './SaleTypeDropdown'
+
+type SearchWidgetProps = {
+    searchParameters: IntersectedSearchAndFilterParams
+    changeSearchParameters: (newParameters: IntersectedSearchAndFilterParams) => void
+}
+
+export function SalesEvidenceSearchWidget (props: SearchWidgetProps) {
+    return (
+        <>
+            <SearchWidget changeSearchParameters={props.changeSearchParameters} searchParameters={props.searchParameters} />
+            <form className="wrapper">
+                <label className="eight">Land Area m2<br />
+                    <input className="minMaxSize border" id="landAreaMin" onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                        props.changeSearchParameters({
+                            ...props.searchParameters,
+                            landArea: { min: e.target.valueAsNumber, max: props.searchParameters.landArea.max }
+                        })
+                    }} placeholder="Default 0" type="number" tabIndex={6} />
+                    <input className="minMaxSize border" id="landAreaMax" onChange={(e: ChangeEvent<HTMLInputElement>) => {
+                        props.changeSearchParameters({
+                            ...props.searchParameters,
+                            landArea: { min: props.searchParameters.landArea.min, max: e.target.valueAsNumber }
+                        })
+                    }} placeholder="Default 10000" type="number" tabIndex={6} />
+                </label>
+
+                <label className="nine">Build Area (sqm) <br />
+                    <input className="minMaxSize border" id="buildArea" onChange={(e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
Change `id="buildArea"` to `id="buildAreaMin"`.

There are a few more of examples of this happening. Just check the rest to make sure you give id a unique string.